### PR TITLE
Fixes #1462 Changing outfit while mounted to unregistered looktypes.

### LINKED ISSUE #1462: Changing outfits while mounted into a looktype that has no mount sprite crashes the client
How to reproduce:
Mount in any of the outfit except the Gamemaster one then change to gamemaster outfit and it crashes the client for you and people around.
## <bountysource-plugin>

Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/24608366-changing-outfits-while-mounted-into-a-looktype-that-has-no-mount-sprite-crashes-the-client?utm_campaign=plugin&utm_content=tracker%2F392151&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F392151&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Xileck**
this fixes #1462

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -3163,6 +3163,11 @@ void Game::playerChangeOutfit(uint32_t playerId, Outfit_t outfit)
 	if (!player) {
 		return;
 	}
+
```
Comment (WibbenZ) [inline src/game.cpp]: Delete the tab

**Comment by ranisalt** [comment]
Squash your commits first.

Run `git rebase -i master`, keep `pick` on the first commit and replace for `squash` on the second one, then choose the message you want on your commit (you can keep the first one) and `git push -f`.

**Comment by Xileck** [comment]
Removed the tab and squashed

**Comment by WibbenZ** [comment]
Would IMO be better to create a Game.hasOutfit(sex, lookType) function, that way it can be used in Lua (insted of the huge if statment in the talkaction etc) and still be used in C++.
Then if people wish to add "custom / monster" outfits, they can just add a simple array and use isInArray.

Atleast sounds better then having to change 3-4 diffrent arrays / huge if statments each update that contains outfits.

**Comment by ranisalt** [comment]
I actually pushed this same modification some time ago and didn't see this PR.

**Pull request closed without merging**